BigTuna/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Aaargh.ttf</string>
```

Comment:
i pulled down the branch and for some reason the .ttf file doesn't exist

Reply:
Shieeeeee